cves/CVE-2017-5648.yml

```diff
@@ -43,7 +37,14 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  UPDATE While investigating bug 60718, it was noticed that some calls to application
```

Comment:
The beginning part seems similar to the CVE website description. Maybe switch things up in your own words, specifically in this part.